lib/govuk_connect/cli.rb

```diff
@@ -304,19 +286,20 @@ def get_domains_for_node_class(target, environment, hosting)
         ssh_username,
         jumpbox_for_environment_and_hosting(environment, hosting),
       ),
-      "govuk_node_list -c #{target}",
+      remote_command,
     ].join(" ")
 
+    log "debug: running command: #{command}"
     output, status = Open3.capture2(command)
 
     unless status.success?
-      error "error: command failed: #{command}"
+      error "\nerror: command failed: #{command}"
       print_empty_line
       print_ssh_username_configuration_help
       exit 1
     end
 
-    output.split("\n").sort
```

Comment:
Just curious as to why we are dropping the `.sort`?

Reply:
I moved that to the methods which use this one.  I thought it could be confusing having this function, whose name suggests it just captures ssh output, transform that output.

Reply:
👍 aaah yes...missed that sorry.